src/pages/ProtectedRoute/index.js

```diff
@@ -2,7 +2,23 @@ import React from "react";
 import { Navigate, Outlet } from "react-router-dom";
 
 export default function ProtectedRoute() {
-  const isAuthUser = localStorage.getItem("authToken");
+  const getCookie = (cookieName) => {
+    const name = `${cookieName}=`;
+    const decodedCookies = decodeURIComponent(document.cookie);
+    const cookies = decodedCookies.split("; ");
+
+    let matchedCookie = "";
```

Comment:
```suggestion
    let matchedCookie = false;
```

matchedCookie는 boolean 값 인가요???? 타입을 맞추는게 좋아보일듯 하네요